Index: ADApp/mar345Src/Makefile
===================================================================
--- ADApp/mar345Src/Makefile    (revision 15692)
+++ ADApp/mar345Src/Makefile    (working copy)
@@ -28,6 +28,7 @@
 PROD_LIBS += mar345
 
 include $(TOP)/ADApp/commonDriverMakefile
+$(PROD_NAME)_DBD += drvAsynIPPort.dbd
 
 #=============================
 
Index: ADApp/pilatusSrc/Makefile
===================================================================
--- ADApp/pilatusSrc/Makefile   (revision 15686)
+++ ADApp/pilatusSrc/Makefile   (working copy)
@@ -27,6 +27,7 @@
 PROD_LIBS += cbfad
 
 include $(TOP)/ADApp/commonDriverMakefile
+$(PROD_NAME)_DBD += drvAsynIPPort.dbd
 
 #=============================
 
Index: ADApp/marCCDSrc/Makefile
===================================================================
--- ADApp/marCCDSrc/Makefile    (revision 15690)
+++ ADApp/marCCDSrc/Makefile    (working copy)
@@ -26,6 +26,7 @@
 PROD_LIBS += marCCD_AD
 
 include $(TOP)/ADApp/commonDriverMakefile
+$(PROD_NAME)_DBD += drvAsynIPPort.dbd
 
 #=============================
 
Index: ADApp/PSLSrc/Makefile
===================================================================
--- ADApp/PSLSrc/Makefile       (revision 15702)
+++ ADApp/PSLSrc/Makefile       (working copy)
@@ -29,6 +29,7 @@
 PROD_LIBS += PSL
 
 include $(AREA_DETECTOR)/ADApp/commonDriverMakefile
+$(PROD_NAME)_DBD += drvAsynIPPort.dbd
 
 #=============================
 
Index: ADApp/brukerSrc/Makefile
===================================================================
--- ADApp/brukerSrc/Makefile    (revision 15696)
+++ ADApp/brukerSrc/Makefile    (working copy)
@@ -25,6 +25,7 @@
 PROD_LIBS += BISDetector
 
 include $(TOP)/ADApp/commonDriverMakefile
+$(PROD_NAME)_DBD += drvAsynIPPort.dbd
 
 #=============================

